libraries/provider_configure.rb

```diff
@@ -91,14 +103,21 @@ def action_manage
     params[:MAX_MAP_COUNT] = new_resource.max_map_count
     params[:ES_JVM_OPTIONS] = "#{new_resource.path_conf}/jvm.options"
 
+    params = params.merge(new_resource.environments) if new_resource.environments
+
+    parameters = {}
```

Comment:
This feels a little convoluted. Is there a reason we're copying params to parameters? And what does environments do?